protocol/message.go

```diff
@@ -30,6 +31,13 @@ type Message struct {
 	// routes that match the filters
 	Filters map[string]string
 
+	// Expires field specifies until when the message is valid to be processed
+	// If this field is set and the message is expired the connectors should
+	// consider the message as processed and log the action
+	//
+	// RFC3339 format
+	Expires *time.Time
```

Comment:
You might be correct, but we had some issues with timezone information checking on other project and would like for the producer to be able to fully control without any uncertainty.